Scrum 48 add and configure prettier via prettierrc

Methika1234 opened the pull request:
Closes: SCRUM48

Methika1234 committed:
style: add Prettier configuration (.prettierrc & .prettierignore)

prdai commented on .github/workflows/bruno-tests.yml:
```diff
@@ -0,0 +1,51 @@
+
+on:
+  push:
+    branches: [main, develop]
```
we do not have a develop since we are not going with a specific structure

prdai commented on .github/workflows/bruno-tests.yml:
```diff
@@ -0,0 +1,51 @@
+  push:
+    branches: [main, develop]
+  pull_request:
+    branches: [main, develop]
```
we do not have a develop since we are not going with a specific structure

prdai commented on .github/workflows/docker-publish.yml:
did you set the actions secret

prdai commented on docker-compose.yml:
we do not need this atm

Methika1234 committed:
`Update GitHub Actions and Docker Compose configuration`

- Updated GitHub Actions workflow to only trigger on main branch.
- Removed Docker Compose configuration file.

Methika1234 committed:
`Update GitHub Actions workflow to use Docker Hub and GitHub Container Registry`

Methika1234 committed:
`Update .github/workflows/docker-publish.yml to use GitHub Container Registry by default`

Methika1234 committed:
Merge branch 'main' into SCRUM-48-add-and-configure-prettier-via-prettierrc
